src/content/developers/docs/layer-2-scaling/index.md

```diff
@@ -18,6 +18,61 @@ You should have a good understanding of all the foundational topics. Implementin
 - It can be unnecessarily expensive to use blockchain applications
 - Any updates to scalability should not be at the expense of decentralization of security – layer 2 builds on top of Ethereum.
 
+## Rollups {#rollups}
+
+Rollups are solutions that bundle or "roll up" sidechain transactions into a single transaction and generate a cryptographic proof, known as a SNARK (succinct non-interactive argument of knowledge). Only this proof is submitted to the main chain.
+
+_Sidechains are Ethereum-compatible, independent blockchains._
+
+In other words, rollups mean that all state and execution is handled in sidechains – signature verification, contract execution, etc. The main Ethereum chain (layer 1) only stores transaction data.
+
+Rollup solutions require relayers who have staked a bond in the rollup contract. This incentivises them to relay rollups accurately.
+
+**Useful for:**
+
+- reducing fees for users
+- open participation
+- fast transaction throughput
+
+There are two types of rollups with different security models:
+
+- Zero knowledge: runs computation off-chain and submits a <b>validity proof</b> to the chain
+- Optimistic: assumes transactions are good and only runs computation, via a <b>fraud proof</b>, in the event of a challenge
+
+### Zero knowledge rollups {#zk-rollups}
+
+Zero knowledge rollups, also known as ZK-Rollups, bundle hundreds of transfers off-chain into a single transaction via a smart contract. From the data submitted, the smart contract can verify all of the transfers that are included. This is known as a validity proof.
+
+With a ZK rollup, validating a block is quicker and cheaper because less data is included. You don't need all the transaction data to verify the transaction, just the proof.
+
+The sidechain where ZK rollups happen have optimised a few things to reduce transaction size further. A user signs up and gets an index rather than an address... this reduces a transaction from 32 bytes to just 4 bytes.
+
+### Optimistic rollups {#optimistic-rollups}
+
+Optimistic rollups use a side chain that sits in parallel to the main Ethereum chain. They can offer improvements in scalability because they don't do any computation by default. Instead, after a transaction they propose the new state to mainnet. Or "notarise" the transaction.
+
+As computation is the slow, expensive part of using Ethereum, this can offer up to 10-100x improvements in scalability dependent on the transaction. This number will increase even more with the introduction of the Eth2 upgrade: [shard chains](/en/eth2/#shard-chains). This is because there will be more data available in the event that a transaction is disputed.
```

Comment:
```suggestion
As computation is the slow, expensive part of using Ethereum, this can offer up to 10-100x improvements in scalability dependent on the transaction. This number will increase even more with the introduction of the Eth2 upgrade: [shard chains](/en/eth2/shard-chains). This is because there will be more data available in the event that a transaction is disputed.
```